float32 yaw_error
float32 dist_error